src/components/Checkout/OrderSummary.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from 'react';
+import {
+	Heading,
+	VStack,
+	SimpleGrid,
+	GridItem,
+	Text,
+	Button,
+	Divider,
+} from '@chakra-ui/react';
+import { colors } from '../../util/constants';
+
+import { useStripe, useElements } from '@stripe/react-stripe-js';
+
+const OrderSummary = () => {
+	const stripe = useStripe();
+	const elements = useElements();
+
+	const [message, setMessage] = useState(null);
```

Comment:
Would it be better to have the default as `''`?

Reply:
done